src/main/fc/rc_adjustments.c

```diff
@@ -75,6 +77,9 @@ uint8_t pidAudioPositionToModeMap[7] = {
 
 static pidProfile_t *pidProfile;
 
+static int activeAdjustmentRangeCount = ADJUSTMENT_RANGE_COUNT_INVALID;
+static uint8_t activeAdjustmentRangeArray[MAX_ADJUSTMENT_RANGE_COUNT];
```

Comment:
Maybe it would be worth splitting adjustment into 2 groups by adjustmentCenter ==0 - it will save some more processing ...

Reply:
Done.